linkerd/app/gateway/src/tests.rs

```diff
@@ -146,4 +161,22 @@ impl Test {
         bg.await?;
         Ok(rsp)
     }
+
+    async fn run_default_profile(self) -> Result<http::Response<http::BoxBody>, Error> {
+        let target = self.target.clone();
+        let suffix = self.suffix.clone();
```

Comment:
Clippy is complaining about this. Since this is `&'static str` you can just `Copy` the reference without cloning:

```suggestion
        let suffix = self.suffix;
```